src/app/dataset/depublication/depublication.component.html

```diff
@@ -21,21 +21,35 @@
     <li *ngIf="isSaving">
       <span class="svg-icon-spin showing"></span>
     </li>
-    <li>
-      <a (click)="onDepublishRecordIds()" [attr.disabled]="depublicationSelections.length === 0">{{
-        'depublishRecordsIds' + (depublicationSelections.length === 1 ? 'Single' : 'Plural')
-          | translate
-      }}</a>
-    </li>
-    <li>
-      <a (click)="onDepublishDataset()" [attr.disabled]="depublishDatasetDisabled()">{{
-        'depublishDataset' | translate
-      }}</a>
+    <li class="dropdown dropdown-options depublish">
+      <a (click)="toggleMenuOptionsDepublish()">Depublish</a>
+      <div *ngIf="optionsOpenDepublish" class="dropdown-wrapper">
+        <ul class="dropdown-content depublish">
+          <li>
+            <a
+              (click)="onDepublishRecordIds()"
+              [attr.disabled]="depublicationSelections.length === 0"
+              >{{
+                'depublishRecordsIds' +
+                  (depublicationSelections.length === 1 ? 'Single' : 'Plural') | translate
+              }}</a
+            >
+          </li>
+          <li>
+            <a (click)="onDepublishRecordIds()">{{ 'depublishRecordsIdsAll' | translate }}</a>
```

Comment:
Should this not be `(click)="onDepublishRecordIds(true)"`? The boolean parameter seems to be missing?

Reply:
Ah, one more thing: Perhaps this menu option (as well as the one above) should also have `depublishDatasetDisabled` as disabled condition (just like the entire dataset option). The same rules apply, after all.

Reply:
I've removed the condition from the item and added it to the entire menu